_benchmark/reference/workloads/test-procedures.md

```diff
@@ -0,0 +1,193 @@
+              "operation-type": "bulk",
+              "bulk-size": 1000
+            },
+            "clients": 8
+          },
+          {
+            "name": "parellel-task-2",
+            "operation": {
+              "operation-type": "bulk",
+              "bulk-size": 500
+            },
+            "clients": 8
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+#### Options
+
+The `parallel` element supports all `schedule` parameters, in addition to the following:
+
+`tasks` | Yes | Array | Defines a list of tasks that should be executed concurrently. 
+`completed-by` | No | String | Allows you define the name of one task in the tasks list, or the value `any`. If a specific task name has been provided then as soon as the named task has completed, the whole parallel task structure is considered completed. If the of value `any` is provided, then any task that completes first renders all other tasks specified in parallel structure complete. If this property is not explicitly defined, the parallel task structure is considered completed as soon as the tasks in the element complete.
+
+### Iteration-based options
+
+Iteration-based options determine the number of times an operation should fun. It can also define the number of iterative runs when tasks are run in [parallel](#parallel-tasks). To configure an iteration-based schedule, use the following options.
+
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`iterations` | No | Integer | Defines a default value for all tasks of the parallel element. Default is `1`.
+`warmup-iterations` | No | Integer | Number of iterations that each client should execute to warmup the benchmark candidate. Warmup iterations will not show up in the measurement results. Default is `0`.
+
+### Time-based options
+
+Time-based options determines the duration of time, in seconds, that operations should run for. This is ideal with batch-style operations which may require an additional warmup period, including batch style operations. 
+
+To configure a time-based schedule, use the following options.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`time-period` | No | Duration | The time period in seconds that OpenSearch Benchmark considers for measurement. Usually not required for bulk indexing, since OpenSearch Benchmark will index all documents at according to the `warmup-time-period`.
+`ramp-up-time-period` | No | Integer | Determines the number of clients used at the end of the specified time period in seconds, which can help increase load gradually. This prevents load spikes from occurring before the benchmark is warmed up. This property requires a `warmup-time-period` to be set as well, which must be less then the ramp up time period. Default is `0`.
+`warmup-time-period` | No | Integer | The time period in seconds to warmup of the benchmark candidate. All response data captured during the warmup period will not appear in the measurement results.
```

Comment:
`The time period in seconds to warmup of the benchmark candidate.` -> `The time period in seconds to warmup the benchmark candidate.`